README.md

```diff
@@ -21,6 +21,7 @@ library to compile the native extensions.
 ### OSX
 
     xcode-select --install
+    rvm install 1.9
```

Comment:
Well, I use rbenv ;) I'm fine with the rvm dependency if 1.9 is necessary, but then we would need to explain how to install rvm.